modules/templates/helper.go

```diff
@@ -261,3 +257,10 @@ func Eval(tokens ...any) (any, error) {
 	n, err := eval.Expr(tokens...)
 	return n.Value, err
 }
+
+func UserThemeName(user *user_model.User) string {
+	if user == nil || user.Theme == "" {
+		return setting.UI.DefaultTheme
+	}
+	return util.Iif(webtheme.IsThemeAvailable(user.Theme), user.Theme, setting.UI.DefaultTheme)
```

Comment:
Why not regular `if` here?

Reply:
Regular `if` is pretty long to me: 52dcda5ce554ca6944fd39a54d068dee1cbf27dc